modules/renter.go

```diff
@@ -349,6 +350,10 @@ type Renter interface {
 	// blocking, including downloads of `offset` and `length` type.
 	DownloadAsync(params RenterDownloadParameters) error
 
+	// ClearDownloadHistory clears the download history of the renter
+	// inclusive for before and after times.
+	ClearDownloadHistory(before, after time.Time) error
```

Comment:
the order of these parameters is confusing to me, because `before` is more recent than `after`, whereas normally you specify a range of time with the older time first (e.g. you say "From 2017 to 2018," not the other way around).

Reply:
That makes sense, I can switch them.